assets/app/views/edit/build-config.html

```diff
@@ -578,6 +578,27 @@
                           </div>
                         </dl>
                       </div>
+                      <div class="section">
+                        <h3>Run Policy
+                          <span class="help action-inline">
+                            <a href>
+                              <i class="pficon pficon-help" data-toggle="tooltip" aria-hidden="true" data-original-title="The build run policy describes the order in which the builds created from the build configuration should run."></i>
+                            </a>
+                          </span>
+                        </h3>
+                        <div class="form-group">
+                          <select
+                            class="form-control"
+                            ng-model="updatedBuildConfig.spec.runPolicy"
+                            ng-options="type as (type | sentenceCase) for type in runPolicyTypes">
+                          </select>
+                        </div>
+                        <div ng-switch="updatedBuildConfig.spec.runPolicy">
```

Comment:
not as concerned about an ng-switch-default here since any empty div isnt really a problem, but you could do something similar here